src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        Random random = new Random();
+        ColorSupplier color = new ColorSupplier();
```

Comment:
```suggestion
    private final Random random = new Random();
    privat final ColorSupplier color = new ColorSupplier();
    
    public Ball getRandomBall() {
        Ball ball = new Ball();
```

Reply:
[random & color objs creating shifted out of meth](https://github.com/Maksym-Kulakov/jv-lottery/commit/2002c128d8a8105d67f0a371c17014e4a4f11a57)